third_party/xla/xla/pjrt/gpu/BUILD

```diff
@@ -211,7 +212,7 @@ cc_library(
 xla_cc_test(
     name = "se_gpu_pjrt_compiler_test",
     srcs = if_gpu_is_configured(["se_gpu_pjrt_compiler_test.cc"]),
-    tags = tf_cuda_tests_tags(),
+    tags = tf_cuda_tests_tags() + ["no_rocm"],
```

Comment:
may I ask why `se_gpu_pjrt_compiler_test `is disable? it's PASSED on XLA upstream side, TF should be ok to enable it.